pkg/util/common.go

```diff
@@ -70,3 +78,75 @@ func CreateDir(d string) error {
 
 	return nil
 }
+
+func GetKey(secrets map[string]string, options map[string]string, keyStoragePath string) (string, error) {
+	if _, err := os.Stat(keyStoragePath); os.IsNotExist(err) {
+		os.Mkdir(keyStoragePath, 0700)
+	}
+
+	keyContents, keyNameExists := secrets["key"]
+	if !keyNameExists {
+		return "", status.Errorf(codes.Internal, "Secret '%s' is unavailable", "key")
+	}
+
+	klog.V(5).Info("Saving key contents to a temporary location")
+	keyFile, err := CreateFile(keyStoragePath, keyContents)
+	if err != nil {
+		return "", status.Errorf(codes.Internal, "Unable to save secret 'key' to %s", keyStoragePath)
+	}
+
+	allFlags := []string{fmt.Sprintf("--key-file=%s", keyFile), "-o=allow_other"}
+	if flags, exists := options["gcs.csi.ofek.dev/flags"]; exists {
```

Comment:
We got rid of this prefix for static way right?

```suggestion
	if flags, exists := options["flags"]; exists {
```

Reply:
That code was there by accident, I'll remove it. (copy-past error)